### DISCUSSION TITLE
menu entry swapper: no option for zanaris

### DISCUSSION THREAD

**Discussion by dani285e** [runelite/runelite#17165] [Ideas]
When using the option to change what right click and shift right click does, you can't choose to set the option to teleport to zanaris.

**Comment by Felanbird** [comment]
Use this 
![image](https://github.com/runelite/runelite/assets/41973452/2f6b848e-d7aa-4687-90db-0527b1d7a7ec)

**Reply by dani285e** [reply]
thanks
